src-tauri/src/main.rs

```diff
@@ -327,9 +327,23 @@ async fn install_northstar_caller(
     window: tauri::Window,
     game_path: String,
     northstar_package_name: Option<String>,
+    version_number: Option<String>,
 ) -> Result<bool, String> {
     log::info!("Running");
-    match install_northstar(window, &game_path, northstar_package_name).await {
+
+    // Get Northstar package name (`Northstar` vs `NorthstarReleaseCandidate`)
+    let northstar_package_name = match northstar_package_name {
```

Comment:
the match case can be replaced with `map` and `unwrap_or` ;)

Reply:
So like

```rust
    let northstar_package_name = northstar_package_name
        .map(|name| {
            if name.len() <= 1 {
                "Northstar".to_string()
            } else {
                name
            }
        })
        .unwrap_or("Northstar".to_string());
```
? ^^"

Reply:
yeah

Reply:
Done in 05483e672b95c664abb1a4df1918006c3b956ebb